admin_guide/out_of_resource_handling.adoc

```diff
@@ -12,39 +12,161 @@ toc::[]
+If swap memory is enabled for a node, that node will not be able to detect that it is under
 *MemoryPressure*.
 
 To take advantage of memory based evictions, operators must
 xref:../admin_guide/overcommit.adoc#disabling-swap-memory[disable swap].
 ====
 
+
 [[out-of-resource-eviction-policy]]
-== Eviction Policy
+== Configuring Eviction Policies
+
+An _eviction policy_ allows a node to fail one or more pods when the node is running low on available resources. 
+Failing a pod allows the node to reclaim needed resources.  
+
+An eviciton policy is a combination of an xref:out-of-resource-eviction-signals[eviction trigger signal] with a specific xref:out-of-resource-eviction-thresholds[eviction threshold value], that is set
+in the xref:out-of-resource-eviction-policy[node configuration file] or through the xref:out-of-resource-eviction-policy-cli[node command line]. Evictions can be either hard, where a node takes immediate action on a pod that exceeds a threshold, or soft, where a node allows a grace period before taking action. Review the sections below for important information the differences between xref:out-of-resource-eviction-thresholds[hard and soft evictions].
+
+By using well-configured eviction policies, a node can proactively monitor for and prevent
+against total starvation of a compute resource. 
+
+For more information on eviction trigger signals, see xref:out-of-resource-eviction-signals[Understanding Eviction Signals] below.
 
-Using eviction policies, a node can proactively monitor for and prevent
-against total starvation of a compute resource.
+For more information on eviction thresholds, see xref:out-of-resource-eviction-thresholds[Understanding Eviction Thresholds] below.
 
-In cases where a node is running low on available resources, it can proactively
-fail one or more pods in order to reclaim the starved resource using an eviction
-policy. When the node fails a pod, it terminates all containers in the pod, and
+[NOTE]
+====
+When the node fails a pod, it terminates all containers in the pod, and
 the `PodPhase` is transitioned to *Failed*.
+====
+
+[[out-of-resource-create-config]]
+=== Using the Node Configuration to Create a Policy 
+
+To configure an eviction policy using the node configuration file:
+
+. Edit the node configuration file on a node.
+
+. Add or edit the eviction thresholds under the `eviction-hard` or `eviction-soft` paramters:
++
+For example:
++
+.Sample Node Configuration file for a hard eviction 
+====
+----
+kubeletArguments:
+  eviction-hard: <1>
+  - memory.available<500Mi<2>
+  - nodefs.available<500Mi
+  - nodefs.inodesFree<100Mi
+  - imagefs.available<100Mi
+  - imagefs.inodesFree<100Mi
+----
+
+<1> The type of eviction: Use this parameter for a xref:out-of-resource-hard-eviction-thresholds[hard eviction].
+<2> An eviction threshold based on a specific eviction trigger signal.
+====
++
+.Sample Node Configuration file for a soft eviction
```

Comment:
since we discussed soft eviction as being rather advanced, i am also worried people will take this as preferred config.